x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/boosts/boost_item.tsx

```diff
@@ -32,6 +32,7 @@ export const BoostItem: React.FC<Props> = ({ id, boost, index, name }) => {
       id={id}
       className="boosts__item"
       buttonContentClassName="boosts__itemButton"
+      initialIsOpen={!!boost.newBoost}
```

Comment:
Does this need a unit test/prop assertion written for this, do you think? No worries if you'd rather not, I'm kinda 50/50 myself

Reply:
I can't help laughing about this, after our lengthy discussion recently where you argued against making assertions.

Reply:
This comment was not a required change request, I apologize if it came off that way. I was just asking, and most of the time when I'm asking, it's just to quickly check that it was an intentional choice vs. something that was missed or forgotten. I'm totally okay with it being an intentional choice.

Thanks so much for going above and beyond with writing unit tests for the entire file, Jason.

Reply:
Understood. Since I had considered it, and then you pointed it out as well, I thought that was a good indication that I should just make the change. Thanks.